canonical-web-and-design/ubuntu.com
# Updated k8s/managed and some illustrations

## Done

- Updated k8s/managed and some illustrations
- Updated some common illustrations
- NOTE @anasereijo - we don't seem to have a design and build icon for openstack/generic, only k8s?

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/kubernetes/managed and then...
    - http://0.0.0.0:8001/core/
    - http://0.0.0.0:8001/download/server/thank-you-s390x
    - http://0.0.0.0:8001/embedded/
    - http://0.0.0.0:8001/financial-services/
    - http://0.0.0.0:8001/kubeflow/
    - http://0.0.0.0:8001/kubernetes/
    - http://0.0.0.0:8001/livepatch/
    - http://0.0.0.0:8001/openstack/consulting
    - http://0.0.0.0:8001/openstack/features
    - http://0.0.0.0:8001/openstack/managed
    - http://0.0.0.0:8001/security/
    - http://0.0.0.0:8001/telecommunications/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare and resolve the [copy doc](https://docs.google.com/document/d/1Qv379nYnod9e4jMrsdDND8D7mbPyqELi2lowjkfD-gU/edit#)


## Issue / Card

Fixes #6474 and Fixes #6519
